sig-scalability/README.md

```diff
@@ -10,7 +10,7 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 
 Responsible for answering scalability related questions such as:
 What size clusters do we think that we should support with Kubernetes in the short to medium term? How performant do we think that the control system should be at scale? What resource overhead should the Kubernetes control system reasonably consume?
-For more details about our objectives please review our [Scaling And Performance Goals](https://github.com/kubernetes/community/blob/master/sig-scalability/goals.md)
+For more details about our objectives please review our [Scaling And Performance Goals](https://git.k8s.io/community/sig-scalability/goals.md)
```

Comment:
`[Scaling And Performance Goals](/sig-scalability/goals.md)`

Reply:
The reason for this one is because it's auto generated from sigs.yaml. I don't want to put a relative link there.